content/en/error_tracking/sentry_sdk.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Sentry SDK
+description: Use the Sentry SDK to send errors to Error Tracking.
+further_reading:
+- link: "/error_tracking/manage_data_collection"
+  tag: "Documentation"
+  text: "Manage Data Collection"
+---
+{{< callout url="#" btn_hidden="true" >}}
+Using the Sentry SDK with Error Tracking is in Preview.
+{{< /callout >}}
+
+<div class="alert alert-warning">
+Using the Sentry SDK with Error Tracking helps you migrate to Datadog. However, to get the most out of Error Tracking, it is recommended to use the regular setup. See <a href="/error_tracking/frontend">Frontend Error Tracking</a> and <a href="/error_tracking/backend">Backend Error Tracking</a>.
+</div>
+
+## Overview
+You can use [Sentry SDKs][1] to send your events to Datadog. This offers you a way to start using Error Tracking on existing applications that are monitored using Sentry SDKs.
+
+Setting up the Sentry SDK with Datadog requires a minimal code change to point the SDK to a Datadog Data Source Name (DSN).
+
+[Events][6] are sent to Datadog as logs. Non-error events (messages) also show up as logs. Other item types (traces, attachments, sessions, ...) are not supported.
+
+## Setup
+### Prerequisites
+- Sentry SDK events are sent into Datadog as **logs**. You must have [Error Tracking for Logs][2] enabled for errors to show up in Error Tracking.
```

Comment:
Maybe we mention.

"To enable Error Tracking for Logs for only your Sentry logs, create a rule (link) like this:" and we provide an example

Reply:
Good suggestion! added!